src/base-element.js

```diff
@@ -414,6 +499,26 @@ export class BaseElement {
     return false;
   }
 
+  /**
+   * Ensure that the element is being eagerly loaded.
+   *
+   * Only used for V2 elements.
+   */
+  ensureLoaded() {}
```

Comment:
Depending on how we land on my later comments re. `custom-element.ensureLoaded()`, I'd love to rename this (esp. if it becomes public API)

Reply:
Let's dedupe to that discussion. Again this is very much the close to the `element.loading = eager`. I think that's where we are moving.